### ISSUE TITLE
Bridge not grouping paused sequences correctly

### ISSUE THREAD

**Issue by jetzlstorfer** [keptn/keptn#5154]
When pausing and resuming in Keptn 0.9 the bridge does not show the grouping of the tasks, in fact, there are shown as individual task.

![image](https://user-images.githubusercontent.com/7677092/132011366-8d076942-9b56-43b4-96b5-334a4fc81fbc.png)

**Comment by jetzlstorfer** [comment]
Without pausing, the helm service is shown in its own task "deployment"
![image](https://user-images.githubusercontent.com/7677092/132012062-01ea4d38-9695-4c58-8115-683fb2d974c2.png)

**Comment by jetzlstorfer** [comment]
Did a new run with pausing again and this time the grouping is correct, however, the next sequence is not triggered, despite the fact that it is not paused anymore.
![image](https://user-images.githubusercontent.com/7677092/132013001-89c4a5ac-48cc-4141-97b8-23a8572b5cf8.png)

**Comment by jetzlstorfer** [comment]
After my third try the bridge for my latest delivery sequence is completely empty. Seems there are some issues...
![image](https://user-images.githubusercontent.com/7677092/132013687-9ba42203-2788-4f81-8039-884fe4f5024d.png)

**Comment by jetzlstorfer** [comment]
I identified why the last screen is empty: Keptn was still trying to execute the previous sequences and therefore no events where coming in for the newest sequence that is shown in the screenshot. Once I aborted all open sequences the execution continued. 
However, I couldn't identify this behavior in the UI but instead had to take a look in the logs of the shipyard service. There should be a way to identify why a sequence is not starting directly in the UI.
